Feat/turtlebot3c image (#1)
ubuntu-robotics/turtlebot3c-ubuntu-core: README.md

# turtlebot3c-ubuntu-core
Turtlebot3c Ubuntu Core image

> The image will contain the TurtleBot3c snaps:

Reviewer: ```suggestion
The image contains the TurtleBot3c snaps:
```

- [turtlebot3c-bringup](https://snapcraft.io/turtlebot3c-bringup)
- [turtlebot3c-nav](https://snapcraft.io/turtlebot3c-nav)
- [turtlebot3c-teleop](https://snapcraft.io/turtlebot3c-teleop)

> The image will create a Hotspot called `turtlebot3c`

Reviewer: ```suggestion
The image creates a Hotspot called `turtlebot3c`
```

with the password `turtlebot3c`.

Once connected to the hotspot,

> the robot will be available at `turtelbot3c.local`

Reviewer: ```suggestion
the robot is available on the network as `turtelbot3c.local`.
Mind that you need avahi installed on your remote machine, see e.g. `snap install avahi`.
```

> All the ROS topics are available on the `ROS_MASTER_URI`:

Reviewer: ```suggestion
Remember to set up the `ROS_MASTER_URI` to point to the robot:
```

```
ROS_MASTER_URI=http://turtelbot3c.local:11311/
```

## Build the image

### Build the gadget snap

> Build the gadget snap with snapcraft

Reviewer: ```suggestion
Build the gadget snap with snapcraft:
```
